src/component/Flow.tsx

```diff
@@ -294,6 +304,30 @@ export default class Flow extends React.Component<FlowProps, FlowState> {
         this.resetState();
     }
 
+    /**
+     * Gets a suggested result name based on the current number of waits
+     * in the current definition
+     */
+    public getSuggestedResultName(): string {
```

Comment:
@ericnewcomer what think you about–in the future–shying away from adding methods to component classes that don't either handle a callback that changes state somewhere in the render tree or determine whether or not JSX is rendered? my reasoning is that a method that doesn't read from the component's `this` context is much easier to test (no need to render `Flow`, access it's instance and then test `getSuggestedResultName`). what do you think?

Reply:
Agreed.